finagle-core/src/main/scala/com/twitter/finagle/Stack.scala

```diff
@@ -455,6 +455,11 @@ object Stack {
       Node(this, (prms, next) => Leaf(this, make(next.make(prms))), next)
   }
 
+  class NoOpModule[T](val role: Role, val description: String) extends Module0[T] {
```

Comment:
*I think* you should be fine. There is an implicit conversion from `A => A` function to `Stackable[A]`.